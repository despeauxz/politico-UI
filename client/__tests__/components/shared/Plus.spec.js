import React from 'react';
import Plus from '@components/shared/Plus';
import toJson from 'enzyme-to-json';


describe('<Plus />', () => {
    it('render the Plus component', () => {
        const shallowWrapper = shallow(<Plus />);

        expect(toJson(shallowWrapper)).toMatchSnapshot();
        expect(shallowWrapper.find('Plus')).toBeTruthy();
    });
});
