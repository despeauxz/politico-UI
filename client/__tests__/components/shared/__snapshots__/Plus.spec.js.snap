// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Plus /> render the Plus component 1`] = `
<svg
  className="fill-current h-4 w-4 opacity-50"
  viewBox="0 0 20 20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M11 9h4v2h-4v4H9v-4H5V9h4V5h2v4zm-1 11a10 10 0 1 1 0-20 10 10 0 0 1 0 20zm0-2a8 8 0 1 0 0-16 8 8 0 0 0 0 16z"
  />
</svg>
`;
